src/transaction/transaction_cl.c

```diff
@@ -1325,6 +1327,7 @@ void
 tran_end_libcas_function (void)
 {
   tm_Is_libcas = false;
+  tm_libcas_depth--;
```

Comment:
isn't it necessary to check "tm_libcas_depth > 0" ?

Reply:
I've added assert at 879013d to check it.